geode-cq/src/distributedTest/java/org/apache/geode/cache/query/cq/dunit/CqQueryDUnitTest.java

```diff
@@ -220,17 +220,17 @@ public void run2() throws CacheException {
 
         for (int i = 0; i < regions.length; i++) {
           createRegion(regions[i], factory.createRegionAttributes());
+          if (getRootRegion("root").getSubregion(regions[i]).isEmpty()) {
+            LogWriterUtils.getLogWriter().info("### CreateServer: Region is empty ###");
```

Comment:
switch this to use the new Logger (from LogService.getLogger()) like you did in the perf test. Is this debug logging to help diagnose a problem?

Reply:
Maybe just remove the debug logs entirely?

Reply:
done